.config/nextest.toml

```diff
@@ -14,3 +14,7 @@ retries = 3
 status-level = "skip"
 failure-output = "immediate-final"
 junit = { path = "junit.xml" }
+
+[profile.default.slow-timeout]
+period = "30s"
```

Comment:
Should this be in here? It might be why your unit tests are failing 😄

If we want to cap the unit tests from having long running tests (e.g., those that block), I'd think something like 10 minutes would be a safer upper limit. Likewise, you could probably just add this to the `ci` profile, as that's used for the unit tests.

Reply:
It probably doesn't need to be in _this_ PR, but it should be in something. A recent PR unit test run was still running 4.5 hours later because there were _no_ timeouts. (retrying with 90s...)